fhirspec.py

```diff
@@ -184,10 +187,16 @@ def class_name_for_type_if_property(self, type_name):
             return None
         return self.settings.replacemap.get(classname, classname)
     
-    def class_name_for_profile(self, profile_name):
-        if not profile_name:
+    def class_name_for_profile(self, profile_names):
```

Comment:
Total nitpicking: I'd keep this `profile_name` since it being a list is an irregular occurrence, and keep the `if not profile_name: return None` at the top.